example/src/main/java/com/attentive/example/ExampleApp.java

```diff
@@ -11,10 +12,12 @@ public class ExampleApp extends Application {
     private static final AttentiveConfig.Mode MODE = AttentiveConfig.Mode.PRODUCTION;
 
     public AttentiveConfig attentiveConfig;
+    public AttentiveEventTracker attentiveEventTracker;
 
     @Override
     public void onCreate() {
-        this.attentiveConfig = new AttentiveConfig(ATTENTIVE_DOMAIN, MODE, this);
+        this.attentiveConfig = new AttentiveConfig(ATTENTIVE_DOMAIN, MODE, getApplicationContext());
+        this.attentiveEventTracker = new AttentiveEventTracker(attentiveConfig);
```

Comment:
Instead of constructing a new EventTracker, it may be better to create a singleton. The Singleton would be easier for the consuming app because they could just call `AttentiveEventTracker.getInstance()` to get the tracker, instead of needing to create a public variable on the `Application` object.

Reply:
this makes sense to me!